src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(getRandomColor(), random.nextInt(MAX_NUMBER));
```

Comment:
lets make random a class-level variable to avoid creating a new instance each time this method is invoked